nautobot/data_validation/models.py

```diff
@@ -177,6 +179,8 @@ class MinMaxValidationRule(ValidationRule):
 
     clone_fields = ["enabled", "content_type", "min", "max", "error_message"]
 
+    is_data_compliance_model = False
```

Comment:
Doesn't this need to be added to `RequiredValidationRule` and `UniqueValidationRule` as well below?